subgraph/src/utils/helper.ts

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-underscore-dangle */
+import { BigInt as GraphBN, Bytes, ethereum, Int8 } from "@graphprotocol/graph-ts";
+
+import { Account, MACI, User } from "../../generated/schema";
+
+import { DEFAULT_MACI_ID } from "./constants";
+
+export const packPubkey = (pubkeyX: GraphBN, pubkeyY: GraphBN): string => `${pubkeyX.toString()}-${pubkeyY.toString()}`;
+
+export const createOrLoadMACI = (
+  event: ethereum.Event,
+  owner: Bytes | null = null,
+  stateTreeDepth: Int8 = 10,
+): MACI => {
+  let maci = MACI.load(DEFAULT_MACI_ID);
+
+  if (!maci) {
+    maci = new MACI(DEFAULT_MACI_ID);
+    maci.owner = owner !== null ? owner : (event.transaction.from as Bytes);
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    maci.stateTreeDepth = stateTreeDepth as i32;
```

Comment:
Why do you need to disable it here and I don't see i32 import in this file.

Reply:
It is required for typecasting in AssemlyScript, but I will change BigInt, cast the type instead